client/parties/actions.ts

```diff
@@ -0,0 +1,121 @@
+import { PartyPayload, PartyUser } from '../../common/parties'
+import { BaseFetchFailure } from '../network/fetch-action-types'
+
+export type PartiesActions =
+  | InviteToPartyBegin
+  | InviteToPartySuccess
+  | BaseFetchFailure<'@parties/inviteToParty'>
```

Comment:
Vastly, vastly prefer that you name these and put them next to the relevant Success type